.gitlab-ci.yml

```diff
@@ -15,8 +15,9 @@ variables:
 default:
   before_script:
     - apk add --no-cache --upgrade git bash findutils grep curl jq shfmt rsync diffutils gawk
+    - git config --global --add safe.directory "*" # fixme?
```

Comment:
What is this for?

Reply:
GitLab CI complained about an inconsistent repo state, so this was a quick fix. I couldn't figure out the reason yet.